### DISCUSSION TITLE
I can't connect any LLM's to a conversation chain

### DISCUSSION THREAD

**Discussion by redmage123** [langflow-ai/langflow#403] [Q&A]
Hello all 

I've been trying to do some simple examples connecting the OpenAI LLM to a conversation chain through langflow.   Unfortunately, I get the following error: 

ValueError: Required input llm for module.  ConversationChain not found   

I've tried multiple combinations of LLMs and other chains and nothing works. 
I'm stuck here.  Any assistance would be appreciated. 

Thanks

**Comment by decentropy** [comment]
I get similar error with just a simple flow: 
HuggingFaceHub, PromptTemplate, LLMChain

Error: "Required input llm for module LLMChain not found"

**Comment by carlosrcoelho** [comment]
Hey!

Thanks for sharing your question here. I'm sorry for not getting back to you sooner.
Langflow has gone through some major changes since then. I recommend trying out the new version for the best experience. If you have any issues, please open a new one; we'll be here to help.
